integrations/src/test/resources/io/jenkins/plugins/casc/SeedJobTest_withGiteaOrganisation.yml

```diff
@@ -0,0 +1,11 @@
+jobs:
+  - script: >
```

Comment:
is there a reason you aren't testing the complete version from the demo folder?

Reply:
I thought it would be cleaner to just test Gitea related stuff. But I can do the whole sample as well.
But first I need to sort out how to get the test pass. Having troubles with a dependent but not loaded plugin.

Reply:
which is good, but there's quite a few more gitea sections in the other one.
but it's fine, if you can get it working we can merge

Reply:
The plan is to add the other Gitea config sections, too. I'll set the PR as draft for now.

Reply:
Hi @timja. Thanks you for reviewing this PR so far. I think I need some help to get the test pass. When executing the `GiteaOrganisationFolderTest` test class, the `job-dsl` plugin fails saying that the `branch-api` plugin does not match the minimum required version (https://github.com/jenkinsci/configuration-as-code-plugin/runs/4341874964?check_suite_focus=true#step:5:556). Within the test suite it is already added to the test scope. So it should be loaded automatically. I've tried to set the version to the latest one and it doesn't work either, which confuses me even more.

Did I miss something about loading plugins inside the test suite? It seems that the `branch-api` plugin is not loaded at all or failed silently while loading. I was able to limit the error cause to the `organizationFolder` declaration (at least I think so). If this is present, the test fails due to plugin requirements. If I replace it with, let's say freestyle job definition, it doesn't complain. I have no idea why this happens and would be happy for any hint.

Reply:
It's https://issues.jenkins.io/browse/JENKINS-48885

Work around may be to use a `RealJenkinsRule` but may need some changes to our rules here as well to make it work. 

You could also `@Ignore` the test for now with a reference to the issue

Reply:
Oh. That's unfortunate. I'll give `RealJenkinsRule` a try. Otherwise I `@Ignore` the test. Thank you for your analysis.